pkg/apis/v1alpha5/register.go

```diff
@@ -25,7 +25,6 @@ const (
 	Group              = "karpenter.sh"
 	ExtensionsGroup    = "extensions." + Group
 	CompatabilityGroup = "compatibility." + Group
-	TestingGroup       = "testing." + Group // Exclusively used for labeling/discovery in testing
```

Comment:
Is there a reason why we're removing this variable, only to use the raw value in the files it was already used in?

Reply:
Actually, good point. We shouldn't remove it from the previous API package (since someone could be relying on it) but we don't want to support this as a native concept moving forward.